packages/jest-preset-mc-app/README.md

```diff
@@ -66,22 +66,21 @@ yarn add \
   jest-enzyme
 ```
 
-2. In your Jest config, instead of importing the preset `@commercetools-frontend/jest-preset-mc-app` you need to import `@commercetools-frontend/jest-preset-mc-app/enzyme/jest-preset-for-enzyme-16.js`
+2. In your Jest config, instead of importing the preset `@commercetools-frontend/jest-preset-mc-app` you need to use the helper function `@commercetools-frontend/jest-preset-mc-app/enzyme/create-jest-preset-with-enzyme.js` to merge one of the main presets with the opt-in Enzyme setup.
 
 For example:
 
 ```js
 const jestPreset = require('@commercetools-frontend/jest-preset-mc-app');
-const jestPresetForEnzyme16 = require('@commercetools-frontend/jest-preset-mc-app/enzyme/jest-preset-for-enzyme-16');
+const createJestPresetWithEnzyme = require('@commercetools-frontend/jest-preset-mc-app/enzyme/create-jest-preset-with-enzyme');
 
 module.exports = {
-  preset: {
-    ...jestPreset,
-    ...jestPresetForEnzyme16,
-  },
+  ...createJestPresetWithEnzyme(16, jestPreset),
```

Comment:
Should we do named object arguments? I mean I know what `16` is but can I do `23`? :)

Reply:
You can try but an error will be thrown that the file does not exist.

How would you expect this to be with named arguments?

Reply:
Something like. `createJestPresetWithEnzyme({ enzypeAdapterVersion: 16, jestPreset })`